docs/source/examples/example_twitch.rst

```diff
@@ -0,0 +1,320 @@
+
+   /usr/local/Caskroom/miniconda/base/envs/datajudge/lib/python3.10/site-packages/datajudge/pytest_integration.py:25: AssertionError
+   _________________ test_func[UniquesEquality::public.twitch_v1 | public.twitch_v2] __________________
+
+   constraint = <datajudge.constraints.uniques.UniquesEquality object at 0x108087e20>
+   datajudge_engine = Engine(postgresql://datajudge:***@localhost:5432/datajudge)
+
+   @pytest.mark.parametrize(
+   "constraint", all_constraints, ids=Constraint.get_description
+   )
+   def test_constraint(constraint, datajudge_engine):
+   test_result = constraint.test(datajudge_engine)
+   >       assert test_result.outcome, test_result.failure_message
+   E       AssertionError: tempdb.public.twitch_v1's column(s) 'language' doesn't have the element(s) '{'Sw3d1zh'}' when compared with the reference values.
+
+   /usr/local/Caskroom/miniconda/base/envs/datajudge/lib/python3.10/site-packages/datajudge/pytest_integration.py:25: AssertionError
+   ___________________ test_func[NumericMean::public.twitch_v2 | public.twitch_v2] ____________________
+
+   constraint = <datajudge.constraints.numeric.NumericMean object at 0x108085a80>
+   datajudge_engine = Engine(postgresql://datajudge:***@localhost:5432/datajudge)
+
+   @pytest.mark.parametrize(
+   "constraint", all_constraints, ids=Constraint.get_description
+   )
+   def test_constraint(constraint, datajudge_engine):
+   test_result = constraint.test(datajudge_engine)
+   >       assert test_result.outcome, test_result.failure_message
+   E       AssertionError: tempdb.public.twitch_v2's column(s) 'average_viewers' has mean 4970.2188235294117647, deviating more than 0.1 from tempdb.public.twitch_v2's column(s) 'average_viewers''s  3567.6584158415841584. Condition on second table: WHERE mature IS TRUE;
+
+   /usr/local/Caskroom/miniconda/base/envs/datajudge/lib/python3.10/site-packages/datajudge/pytest_integration.py:25: AssertionError
+   ===================================== short test summary info ======================================
+   FAILED twitch_specification.py::test_func[VarCharRegex::tempdb.public.twitch_v2] - AssertionError...
+   FAILED twitch_specification.py::test_func[KolmogorovSmirnov2Sample::public.twitch_v1 | public.twitch_v2]
+   FAILED twitch_specification.py::test_func[UniquesEquality::public.twitch_v1 | public.twitch_v2]
+   FAILED twitch_specification.py::test_func[NumericMean::public.twitch_v2 | public.twitch_v2] - Ass...
+   =================================== 4 failed, 4 passed in 1.80s ====================================
+
+So we see that we might not want to trust version 2 of the data as is. What exactly do we
```

Comment:
```suggestion
So we see that we might not want to blindly trust version 2 of the data as is, and that a new exploration might be in order. What exactly do we
```

Reply:
I was unfamiliar with this usage of the expression 'in order' and took the liberty to rephrase slightly.